src/services/authentication/authentication.service.ts

```diff
@@ -85,4 +85,31 @@ export async function confirmAccountService(token: string) {
     if(!updatedUser) throw new UserCouldNotBeAuthenticated('User could not be updated');
     accessLogger.info('Trying to confirm account', { token });
     return token;
-}
\ No newline at end of file
+}
+
+export const resetPasswordMailService = async (email: string) => {
+    accessLogger.info("Checking user for email", { email });
+    const user = await oneUserByEmail({ email });
+    if (!user) throw new UserCouldNotBeAuthenticated('User could not be found');
+    const token = createToken(user);
+    try {
+        await sendResetPasswordEmail(email, createMailingURL(token, Routes.RESET_PASSWORD));
+    } catch(e) {
+        console.log(e);
+        throw new Error("Could not send email");
+    }
+};
+
+export const resetPassword = async (token: string, password: string): Promise<string> => {
```

Comment:
Ah j'ai compris faudra renommer resetPassword en resetPasswordService
```suggestion
export const resetPasswordService = async (token: string, password: string): Promise<string> => {
```